### ISSUE TITLE
Editor not working with WonderCMS 2.3.1

### ISSUE THREAD

**Issue by robiso** [Hermadme/Medium-Editor_wonderCMS_2.x.x#1]
Don't know what's causing the issue, but the editor doesn't work (even with manual installation), the toolbar is shown but no editing is possible.

Additional note:
The latest version of WonderCMS 2.3.1 has a built in plugin installer. When using your release ZIP link, it incorrectly unzips your plugin.

Here's an example of a correctly zipped mediumeditor_wCMS (just contains a single folder, as the latest version of WonderCMS extracts the ZIP URL the users paste in their settings panel.
[mediumeditor_wCMS.zip](https://github.com/Hermadme/Medium-Editor_wonderCMS_2.x.x/files/1252781/mediumeditor_wCMS.zip)
(if you copy paste this in your settings panel, it will correctly install it).
But like mentioned earlier, something is causing it to not work properly.

So in short:
- no idea why the editor is breaking
- a new release with mediumeditor_wCMS zipped without any additional folders will make it easy for users to install your plugin within their Settings->Themes & plugins
